NEWS.md

```diff
@@ -1,5 +1,48 @@
 # OSBuild - Build-Pipelines for Operating System Artifacts
 
+## CHANGES WITH 20:
+
+  * The filesystem assemblers gained support for btrfs. They can
+    now output image files as btrfs, similar to the existing support
+    for ext4 and xfs.
+
+  * The `--libdir=DIR` handling was generalized in that an empty
+    `osbuild` subdirectory will now always cause osbuild to use the
+    system osbuild package. This means a custom `libdir` via
+    `--libdir=DIR` no longer requires the entire osbuild python
+    package to be bundled in an `osbuild` subdirectory.
+
+  * When run on a terminal, `osbuild` will now output the duration
+    of a stage (or other module).
+
+  * The `--output-directory` switch is now mandatory if no checkpoint
+    was specified. In this situation, running `osbuild` would be a
+    no-op.
+
+  * The `ostree` assembler now optionally emits version metadata in
+    its commits.
+
+  * `osbuild` now supports running on Ubuntu-20.04.
+
+  * Modules can now pass metadata alongside the filesystem objects
+    they emit. This metadata is not stored in the final artifact, but
+    passed to the caller via the structured osbuild output.
+
+  * The `ostree` assembler now emits compose metadata as part of its
+    build. This can be inspected by the caller to get detailed compose
+    information.
+
+  * The `rpm` stage now emits detailed metadata about the installed
+    RPM packages.
+
+  * Lots of fixes all over the place, including SELinux reworks and
+    PEP-8 conformance.
+
+Contributions from: Christian Kellner, David Rheinsberg, Davide Cavalca,
+                    Major Hayden, chloenayon
```

Comment:
@chloenayon might wanna change her git user name :D

Reply:
I have edited these manually in the past, given the email and github information. Then people blamed me for changing their intended username. Since then, I just take whatever people give me in their git-username.

In this case, I leave it to Chloe to decide ;)

Reply:
Yeah, I did it once, but lets leave it like this for now. Can also fix it later